src/main/resources/application.properties

```diff
@@ -102,6 +102,9 @@ elasticsearch.channel.index = channelfinder
 # maximum query result size
 elasticsearch.query.size = 10000
 
+# WARNING this changes the elastic settings. UPDATE  with care.
+elasticsearch.maxResultWindow.size = 10000
```

Comment:
You should update the test application.properties.

Reply:
sure